src/scheduler/globals.h

```diff
@@ -50,7 +50,15 @@ extern "C" {
 #include "sched_cmds.h"
 
 extern void *poll_context;
-extern ds_queue *sched_cmds;
+/* total number of sched commands as of today */
+#define TOTAL_SCHED_CMDS 18
```

Comment:
Thanks Bhroam. I have created an enum for the sched commands as you suggested. I did not make it an enum earlier because there was a gap between SCH_SCHEDULE_CMD(value 5) and SCH_CONFIGURE(value 7). Number 6 was missing. Don't know the reason for this. But since these are internal numbers used to identify commands I don't think it really matters. Anyways now it is enum. Please look into my latest commits for changes.